src/cozyhelpers.js

```diff
@@ -0,0 +1,21 @@
+const { basename, dirname } = require('path')
+const { log, cozyClient } = require('cozy-konnector-libs')
+
+module.exports = {
+  mkdirp
+}
+
+// create a folder if it does not already exist
+function mkdirp(path) {
```

Comment:
Let's move this to `cozy-client` or `cozy-konnector-lib` when we have the time

Reply:
This was on my shortlist: I mostly copied it from pajemploi where @doubleface put it 😄

Reply:
I'll take a look at `saveBills()`, I just forgot about it.

Reply:
saveBills may be changed to [saveDocuments](https://github.com/konnectors/libs/pull/184) but will still be available for retrocompatibility anyway.

Reply:
- [x] Use `saveBills()` (although I can't test it right now because the Darty backend is down...)

Reply:
**update** Website is back and amount parsing seems to work.